Performance/FlxBunnyMark/source/PlayState.hx

```diff
@@ -35,7 +37,17 @@ class PlayState extends FlxState
 	private var _offScreenButton:FlxButton;
 	private var _bunnyCounter:FlxText;
 	private var _fpsCounter:FlxText;
-
+	
+	#if !flash
```

Comment:
Would be nicer to have a `shaders_supported` haxedef that we define in the `Project.xml`. That way only one place has to be changed if the targets on which we support shaders change.